src/frontend/Komponenter/Behandling/VedtakOgBeregning/Barnetilsyn/UtgiftsperiodeValg.tsx

```diff
@@ -195,49 +218,42 @@ const UtgiftsperiodeValg: React.FC<Props> = ({
                                 {barnForPeriode
                                     .filter((barn) => utgiftsperiode.barn.includes(barn.value))
                                     .map((barn) => (
-                                        <BodyShortSmall>{barn.label}</BodyShortSmall>
+                                        <BodyShortSmall key={barn.value}>
+                                            {barn.label}
+                                        </BodyShortSmall>
                                     ))}
                             </ContainerMedLuftUnder>
                         )}
-                        <AntallBarn lesevisning={behandlingErRedigerbar}>{`${
-                            utgiftsperioder.value[index].barn
-                                ? utgiftsperioder.value[index].barn?.length
-                                : 0
-                        }`}</AntallBarn>
-                        <StyledInput
-                            onKeyPress={tilHeltall}
-                            type="number"
-                            size={'small'}
-                            value={harTallverdi(utgifter) ? utgifter : ''}
-                            disabled={erMidlertidigOpphør}
-                            onChange={(e) => {
-                                oppdaterUtgiftsperiode(
-                                    index,
-                                    EUtgiftsperiodeProperty.utgifter,
-                                    tilTallverdi(e.target.value)
-                                );
-                            }}
-                            erLesevisning={!behandlingErRedigerbar}
-                            label={'Utgifter'}
-                            hideLabel
-                        />
-                        <CheckboxContainer>
-                            {!behandlingErRedigerbar && erMidlertidigOpphør ? (
-                                <TekstIngenStønad>Ja</TekstIngenStønad>
-                            ) : (
-                                <FamilieCheckbox
-                                    erLesevisning={!behandlingErRedigerbar}
-                                    checked={erMidlertidigOpphør}
-                                    onChange={() => {
-                                        oppdaterUtgiftsperiodeDersomMidlertidigOpphør(
-                                            index,
-                                            !erMidlertidigOpphør
-                                        );
-                                    }}
-                                    children={null}
-                                />
-                            )}
-                        </CheckboxContainer>
+                        {midlertidigOpphør ? (
+                            <div />
```

Comment:
Burde vi bruke `grid-template-areas` for å slippe tom div? Kanskje ikke så viktig, men DOM-hadde kanskje blitt mer semantisk